mindsdb/interfaces/database/integrations.py

```diff
@@ -400,14 +400,14 @@ def create_tmp_handler_args(self, handler_type: str, connection_data: dict) -> o
             root_dir='tmp',
             sync=False
         )
-        handler_storage = HandlerStorage(integration_id, root_dir='tmp')
+        handler_storage = HandlerStorage(integration_id, root_dir='tmp', is_temporal=True)
```

Comment:
should `is_temporal` be deleted?

Reply:
with is_temporal the  fileStorage.push_path isn't executed
without it I saw files were copied from 'content' to 'storage' folder localy

Reply:
in that case better will be set `sync` option of file storage in handler storage